src/widgets/histogram/content-view.js

```diff
@@ -139,6 +139,11 @@ module.exports = cdb.core.View.extend({
       this._setupBindings();
       this.$el.toggleClass('is-collapsed', !!this.model.get('collapsed'));
       this._initViews();
+
+      if (!this.model.get('collapsed')) {
```

Comment:
There are no issue, but I agree is not the right way to solve, it simply was the quickest. I will give another look. Thanks !!